### PR TITLE
add docs for order model

### PR DESCRIPTION
**Pull request by zubair-arbi**
ECOM-2236
@awais786 @aamir-khan @ahsan-ul-haq @tasawernawaz 
- Add basic docs for `Order` model in the `order.rst` file.
- Add populated screen shot of an order in the django admin

Code changes (PR In-code-review): https://github.com/edx/ecommerce/pull/329
Preview: https://github.com/edx/ecommerce/blob/zub/story/ecom-2236-add-order-model-docs/docs/order.rst

### CODE REVIEW

**Comment by zubair-arbi** [comment]
@clintonb Please review

**Comment by ahsan-ul-haq** [comment]
:+1:

**Comment by zubair-arbi** [comment]
@srpearce please review

File: docs/order.rst [hunk 1]
```diff
@@ -0,0 +1,21 @@
+======================
+Order Module
```
Comment (srpearce) [inline docs/order.rst:2]: Should this be "Model"?

File: docs/order.rst [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+======================
+
+This model is provided by the ``django-oscar``.
```
Comment (srpearce) [inline docs/order.rst:5]: Add a noun after `django-oscar` (for example, "This model is provided by the `django-oscar` interface/system/correct noun.")

File: docs/order.rst [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+We have customized it in order to add a partner in it.
+
+
+---------------------------------------------
+Order Model | Django Admin
+---------------------------------------------
+
+The order object is automatically created when a user purchases a course from a partner.
+To update the partner of an order, use the ``Order`` model in the `Django administration panel <http://localhost:8002/admin/order/order/>`_.
```
Comment (srpearce) [inline docs/order.rst:14]: I'm not sure what "To update the partner of an order" means - the following are a couple of suggestions for this:

To update a partner's order, use...

To notify the partner that an order has occurred, use...

File: docs/order.rst [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+This panel is located at http://localhost:8002/admin/order/order/{order_id}/ ('order_id' is the id of the order).
```
Comment (srpearce) [inline docs/order.rst:15]: This panel is located at the following URL, where {order_id} is the ID of the order. _(SP: Are there any constraints around the order ID - for example, do all characters have to be alphanumeric, or does it have to be a specific length?)_

http://localhost:8002/admin/order/order/{order_id}/

_(SP: Make sure to indent the URL under the text.)_

**Comment by clintonb** [comment]
Closing as the only documentation needed for the Order model is a mention that it is linked to a Partner. We do not want to encourage anyone to use the Django admin for the order model.

**Pull request closed without merging**